src/org/ods/orchestration/usecase/BitbucketTraceabilityUseCase.groovy

```diff
@@ -107,34 +107,49 @@ class BitbucketTraceabilityUseCase {
         boolean nextPage = true
         int nextPageStart = 0
         while (nextPage) {
-            Map commits = bitbucketService.getCommitsForIntegrationBranch(token, repo.repo, PAGE_LIMIT, nextPageStart)
+            Map pullRequests = bitbucketService.getMergedPullRequestsForIntegrationBranch(token, repo,
+                PAGE_LIMIT, nextPageStart)
```

Comment:
We are going to increase the page limit to 100

Reply:
done